frontend/src/metabase/visualizations/visualizations/Scalar/Scalar.jsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable react/prop-types */
```

Comment:
Converted to a functional component without any functional changes to use `useBrowserRenderingContext` hook